packages/wdio-browserstack-service/src/util.ts

```diff
@@ -44,7 +44,7 @@ export function getBrowserCapabilities(browser: Browser<'async'> | MultiRemoteBr
  * @param cap browser capabilities
  */
 export function isBrowserstackCapability(cap?: Capabilities.Capabilities) {
-    return Boolean(cap && cap['bstack:options'])
+    return Boolean(cap && cap['bstack:options'] && Object.keys(cap['bstack:options']).length && !(Object.keys(cap['bstack:options']).length === 1 && cap['bstack:options'].wdioService))
```

Comment:
Can we make this a bit more readable and also add a comment what you are checking exactly here?
```suggestion
    return (
        Boolean(cap && cap['bstack:options'] &&
        !(
            Object.keys(cap['bstack:options']).length === 1 &&
            cap['bstack:options'].wdioService
        )
    )
```

Reply:
let's add a comment to the code with the explanation

Reply:
Hey, checking further into this issue as found another scenario where this error is thrown (which is independent of the code adding wdioService cap)  - when multiremote is running on Automate + AppAutomate sessions. Will look into the fix and push changes to handle that.